Jetstream/Session.swift

```diff
@@ -33,14 +33,13 @@ public class Session {
     let client: Client
     var nextMessageIndex: UInt = 1
     var serverIndex: UInt = 0
-    var scopes = [UInt: Scope]()
+    var scopes = [UInt: (Scope, [String: AnyObject])]()
```

Comment:
Never mind fixed, weird source character creeped into my file that was invisible